sdk/core/azure-core/tests/testserver_tests/coretestserver/coretestserver/test_routes/streams.py

```diff
@@ -32,8 +32,11 @@ def streaming_test():
 def stream_compressed_header_error():
     yield b'test'
 
-def stream_compress():
-    file_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", "./files/test.tar.gz"))
+def stream_compressed_no_header():
+    try:
+        file_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "..", "./files/test.tar.gz"))
+    except:
+        file_path = os.path.abspath(os.path.join(os.path.abspath(__file__), "..", "./files/test.tar.gz"))
     with open(file_path, "rb") as fd:
         yield fd.read()
```

Comment:
The FileNotFoundError arises from the "with open" code so I think you need to include that in your try/except